operator/bundle_helpers/patch-csv.py

```diff
@@ -155,7 +155,7 @@ def parse_args():
                         help='Which SemVer version of the operator to set in the patched CSV, e.g. 3.62.0')
     parser.add_argument("--first-version", required=True, metavar='version',
                         help='The first version of the operator that was published')
-    parser.add_argument("--operator-image", required=True, metavar='image',
+    parser.add_argument("--operator-image", metavar='image',
```

Comment:
Why make this optional? I don't see a place here where we call this script omitting this flag?